interp/hello_interp/CMakeLists.txt

```diff
@@ -3,12 +3,12 @@ if (TARGET hardware_interp)
             hello_interp.c
             )
 
-    # Pull in dependencies
+    # pull in common dependencies and additional interpolation hardware support
```

Comment:
I dunno if this ought to be "interpolator" rather than "interpolation" ?  (minor nit-pick so feel free to ignore)

Reply:
I agree!